src/backend/launcher.ts

```diff
@@ -447,23 +455,40 @@ function launchCleanup(rpcClient?: RpcClient) {
     logInfo('Stopped Discord Rich Presence', { prefix: LogPrefix.Backend })
   }
 }
-async function runWineCommand(
-  game: Game,
-  command: string,
-  wait: boolean,
-  forceRunInPrefixVerb = false
-) {
-  const gameSettings = await game.getSettings()
-  const { folder_name: installFolderName } = game.getGameInfo()
 
-  const { wineVersion } = gameSettings
+async function runWineCommand({
+  gameSettings,
+  command,
+  wait,
+  forceRunInPrefixVerb,
+  installFolderName
+}: WineCommandArgs) {
+  const settings = gameSettings
+    ? gameSettings
+    : await GlobalConfig.get().getSettings()
+  const { wineVersion, winePrefix } = settings
+
+  if (!existsSync(winePrefix)) {
+    mkdirSync(winePrefix, { recursive: true })
+  }
+
+  if (['exe', 'msi', 'bat'].includes(command.split('.')[1])) {
```

Comment:
should we do this always just in case? no idea which file extensions we could have here but I imagine it's safe to quote any command?

Reply:
Don't need this anymore since I refactored it to use `spawn` now. If we need to deal with spaces so it is better to avoid using `exec` and `execAsync` from now on.